mesa/space.py

```diff
@@ -4,9 +4,12 @@
 
 Objects used to add a spatial component to a model.
 
-Grid: base grid, a simple list-of-lists.
-SingleGrid: grid which strictly enforces one object per cell.
-MultiGrid: extension to Grid where each cell is a set of objects.
+Grid: base grid, it creates a square grid.
+SingleGrid: extension to Grid which strictly enforces one agent per cell.
+MultiGrid: extension to Grid where each cell can contain a set of agents.
+HexGrid: extension to Grid to handle hexagonal neighbors.
+ContinuousSpace: a nondiscrete space where each agent can have an arbitrary position.
```

Comment:
I think it is more precise to mention float position somehow here. But this version is fine too for now.

Reply:
"arbitrary float position"?

Reply:
I was about to say that, but the position is a tuple `tuple[float, float]`.

Reply:
mmh "two dimensional arbitrary float position"?

Reply:
What about "a two-dimensional space where each agent has an arbitrary position of `float`'s"?

Reply:
sounds good, made it that way